### ISSUE TITLE
Add people left out without a pair to other pair and make a match for 3 people

### ISSUE THREAD

**Issue by kelostrada** [kelostrada/chitczat#6]

**Comment by agusmoles** [comment]
What do you think about checking if a person didn't get matched with anyone after the matching algorithm and add it to a random match if noone talked to each other before? Also we should add those 3 matches in the `matches` collection, right?

**Comment by kelostrada** [comment]
Yeah I was thinking that :) Although to not change much maybe not add 3 matches but add two corresponding matches. 

Like A, B, C match together - we create matches: [[A,B], [B,C], [C,A]]. Im not yet sure if this is the correct way but Im thinking this might be the easiest to do (however when displaying matches it might complicate things a little bit)

**Comment by agusmoles** [comment]
> Yeah I was thinking that :) Although to not change much maybe not add 3 matches but add two corresponding matches.
> 
> Like A, B, C match together - we create matches: [[A,B], [B,C], [C,A]]. Im not yet sure if this is the correct way but Im thinking this might be the easiest to do (however when displaying matches it might complicate things a little bit)

I think it will work :thinking: and yes, its the easiest
